modules/custom/activity_viewer/src/Plugin/views/filter/ActivityNotificationVisibilityAccess.php

```diff
@@ -110,6 +110,7 @@ public function query(): void {
 
     if ($account->isAnonymous()) {
       $configuration['table'] = 'node_field_data';
+      $configuration['field'] = 'nid';
```

Comment:
I think we need explicitly add join configuration ([see removed one](https://github.com/goalgorilla/open_social/commit/e2dcb4df212788199e9c8ebb497ff1d2d451b3fa#diff-9f4af291e020bd62b0a06df90dd4e6e8c03359606815f5e446f68087e8adf38dL88))
```php
    $configuration = [
      'left_table' => 'activity__field_activity_entity',
      'left_field' => 'field_activity_entity_target_id',
      'table' => 'node_field_data',
      'field' => 'nid',
      'operator' => '=',
      'extra' => [
        0 => [
          'left_field' => 'field_activity_entity_target_type',
          'value' => 'node',
        ],
      ],
    ];
```